MAUI/DataForm/Validation.md

```diff
@@ -15,6 +15,108 @@ The data form validates the data and user input to update the correct value in t
 
 The supported built in validations are as follows:
 
+#### IDataErrorInfo
+You can validate the data by implementing the [IDataErrorInfo](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.idataerrorinfo?view=net-6.0) interface in the data object class.
+
+{% tabs %}
+{% highlight C# %}
+
+public class EmployeeInfo : IDataErrorInfo
+{
+    public event PropertyChangedEventHandler PropertyChanged;
+
+    public EmployeeInfo()
+    {
+
+    }
+
+    public int EmployeeID { get; set; }
+
+    public string Name { get; set; }
+
+    public string Title { get; set; }
+
+    [Display(AutoGenerateField = false)]
+    public string Error
+    {
+        get
+        {
+            return string.Empty;
+        }
+    }
+
+    [Display(AutoGenerateField = false)]
+    public string this[string columnName]
+    {
+        get
+        {
+            if (columnName == nameof(Title) && this.Title.Contains("Marketing"))
+            {
+                return "Marketing is not allowed";
+            }
+
+            return string.Empty;
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+
+#### INotifyDataErrorInfo
+
+You can validate the data by implementing the [INotifyDataErrorInfo](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.inotifydataerrorinfo?view=net-6.0) interface in the data object class. This interface has three members,
+
+* HasErrors property
+* GetErrors method
+* ErrorsChanged event
+
+{% tabs %}
+{% highlight C# %}
+
+public class EmployeeInfo : INotifyDataErrorInfo
+{
+    public event PropertyChangedEventHandler PropertyChanged;
```

Comment:
public event PropertyChangedEventHandler PropertyChanged;

Is this required, if not remove

Reply:
removed